sessions/recap-project-6/tourio-app/README.md

```diff
@@ -26,9 +26,23 @@ In this project you will build a Tourio App, the best friend of all travelers: i
 - In `pages/places/[id].index.js`, write the `deletePlace` function to start a `DELETE` request.
 - Write the `DELETE` API route in `pages/api/places/[id]/index.js`.
 
-### Deploy to Vercel
+## Bonus
 
-Deploy your app to Vercel and make sure the deployment can access the cloud database with the help of environment variables.
+### Comment Section
+
+- To add a comment section for each place, you'll need to create a database schema or collection to store comments associated with each place. Each comment should have a reference to the place it belongs to, such as a place ID.
+
+- Use the [`lib/db_comments.json` file](lib/db_comments.json) to insert sample data.
+
+- Write a POST API route to handle comment submissions. This route should associate each comment with the appropriate place in your database.
+
+- Implement a mechanism to retrieve, display and delete comments for each place on the detail page.
+
+- Consider the case where a place is deleted. Ensure associated comments are also removed to avoid orphaned data in your database.
+
+### Deploy your Project to Vercel
+
+Follow [these instructions](https://github.com/neuefische/web-protocol-template/blob/main/docs/deployment-vercel.md) to deploy the project to Vercel.   
```

Comment:
@F-Kirchhoff this would mean our students would need to have access to neuefische repo too.

Reply:
you are absolutely correct. We probably need to move the curriculum and exercises into a neutral organisation, so we could move these docs there as well as soon we decided on how we want to share our common resources with the cohorts.